git_machete/client.py

```diff
@@ -1383,34 +1383,41 @@ def is_excluded_reflog_subject(sha_: str, gs_: str) -> bool:
+        return RemoteAndOrganizationAndRepository(remote=self.__git.get_config_attr_or_none(f"machete.github.remote"),
+                                                  organization=self.__git.get_config_attr_or_none(f"machete.github.organization"),
+                                                  repository=self.__git.get_config_attr_or_none(f"machete.github.repository"))
+
     def sync_annotations_to_github_prs(self) -> None:
 
         url_for_remote: Dict[str, str] = {remote: self.__git.get_url_of_remote(remote) for remote in
                                           self.__git.get_remotes()}
         if not url_for_remote:
             raise MacheteException(fmt('No remotes defined for this repository (see `git remote`)'))
 
-        optional_org_name_for_github_remote: Dict[str, Optional[Tuple[str, str]]] = {
-            remote: get_parsed_github_remote_url(url) for remote, url in url_for_remote.items()}
-        org_name_for_github_remote: Dict[str, Tuple[str, str]] = {remote: org_name for remote, org_name in
-                                                                  optional_org_name_for_github_remote.items() if
-                                                                  org_name}
-        if not org_name_for_github_remote:
-            raise MacheteException(
-                fmt('Remotes are defined for this repository, but none of them '
-                    'corresponds to GitHub (see `git remote -v` for details)'))
-
-        org: str = ''
-        repo: str = ''
-        if len(org_name_for_github_remote) == 1:
-            org, repo = list(org_name_for_github_remote.values())[0]
-        elif len(org_name_for_github_remote) > 1:
-            if 'origin' in org_name_for_github_remote:
-                org, repo = org_name_for_github_remote['origin']
-            else:
+        remote: str
+        org: str
+        repo: str
+        remote, org, repo = self.get_remote_name_and_organization_and_repository_name_of_remote()
+        if not(remote and org and repo):
+            optional_org_name_for_github_remote: Dict[str, Optional[RemoteAndOrganizationAndRepository]] = {
+                remote: get_parsed_github_remote_url(url, remote) for remote, url in url_for_remote.items()}
+            org_name_for_github_remote: Dict[str, RemoteAndOrganizationAndRepository] = {remote: org_name for remote, org_name in
+                                                                                         optional_org_name_for_github_remote.items() if
+                                                                                         org_name}
+            if not org_name_for_github_remote:
                 raise MacheteException(
-                    f'Multiple non-origin remotes correspond to GitHub in this repository: '
-                    f'{", ".join(org_name_for_github_remote.keys())}, aborting')
+                    fmt('Remotes are defined for this repository, but none of them '
```

Comment:
(and here 👇🏻)

Reply:
@PawelLipski same situation as above

Reply:
I mean... that's exactly Konrad's case from the original issue 😅 so informing about the config keys would be a good idea

Reply:
Yeah, i see your point now :D